src/Registration/Registration.jsx

```diff
@@ -0,0 +1,124 @@
+    e.preventDefault();
+    // TODO bootstrap validation instead of validation below?
+    if (user.password !== user.confirmPassword) {
+      setUser({
+        ...user,
+        error: "The passwords do not match!",
+      });
+      // Stop the form from submitting
+      return false;
+    }
+
+    firebase
+      .auth()
+      .createUserWithEmailAndPassword(user.email, user.password)
+      .then((userCredential) => {
+        // Set display name for new user
+        userCredential.user.updateProfile({
+          displayName: user.displayName,
+        });
+
+        setCreatedState(true);
+
+        // Sign user out
+        firebase.auth().signOut();
+      })
+      .catch((error) => {
+        const errorMessage = error.message;
+        setUser({
+          ...user,
+          error: errorMessage,
+        });
+        setCreatedState(false);
+      });
+    return true;
+  };
+
+  const handleChange = (e) => {
+    setUser({
+      ...user,
+      [e.target.name]: e.target.value,
+      error: "",
+    });
+    setCreatedState(false);
+  };
+
+  return (
+    <Container fluid="md">
+      <Card>
+        <Card.Title>Create an account</Card.Title>
+        <Form className="registration-form" onSubmit={handleSubmit}>
+          <Form.Group controlId="formDisplayName">
+            <Form.Label>Full Name:</Form.Label>
+            <Form.Control
+              type="text"
+              placeholder="E.g: Joe Johnson"
+              onChange={handleChange}
+              name="displayName"
+              required
+            />
+          </Form.Group>
+          <Form.Group controlId="formEmail">
+            <Form.Label>Email address:</Form.Label>
+            <Form.Control
+              type="email"
+              placeholder="E.g: joe123@gmail.com"
+              className="email"
+              onChange={handleChange}
+              name="email"
+              required
+            />
+          </Form.Group>
+          <Form.Group controlId="formPassword">
+            <Form.Label>Password:</Form.Label>
+            <Form.Control
+              type="password"
+              placeholder="Your Password"
+              className="password"
+              onChange={handleChange}
+              name="password"
+              required
+              min={6}
+            />
+          </Form.Group>
+
+          <Form.Group controlId="formConfirmPassword">
+            <Form.Label>Password confirmation:</Form.Label>
+            <Form.Control
+              type="password"
+              placeholder="Confirm Password"
+              onChange={handleChange}
+              name="confirmPassword"
+              required
+              min={6}
+            />
+          </Form.Group>
+          {user.error && <Alert variant="danger">{user.error}</Alert>}
+          {createdState && (
+            <Alert variant="info">You have created an account!</Alert>
+          )}
+          <Button variant="danger" type="submit">
```

Comment:
`variant="danger" `
Red colored buttons will become more reoccurring on other components, according to our mockups.
I don't think the word "danger" is really applicable for all of them. Maybe it's better if we define our own variant. What do you suggest we should do in the future?

Reply:
I think that could be a nice addition, there should be a difference between the 'danger'-kind of buttons/alerts and the 'normal' red coloured buttons/alerts. I think we can discuss this with the rest of the team.